content/docs/user-guide/gems/reference/debug/imgui.md

```diff
@@ -1,8 +1,102 @@
+
+One of the great things about ImGui is that there is next to no overhead when the ImGui game overlay is not visible, which is the default state. It is also very lightweight while in use. 
+
+## Accessing ImGui menus
+
+ImGui can be accessed by pressing the `HOME` key in the editor, game or server launchers.
+
+![ImGui Menus in Viewport](/images/user-guide/gems/reference/debug/imgui_menus.png)
+
+> Server launchers support ImGui if rendering is active. ImGui requires the `rhi` cvar to not be null.
+
+You can use a keyboard, mouse or controller to navigate the ImGui menus.
+
+### Keyboard navigation
+For keyboard use the arrow keys to navigate the menus and `space` to activate or select an option.
+
+### Controller navigation
+Controller support is disabled by default, to turn it on you can use either the `imgui_EnableController` or `imgui_EnableControllerMouse` [CVARs](#cvars) to simulate the mouse with the controller.
+
+If using the controller then the following inputs are supported:
+
+| Input                 | Description                                                                                                       | 
+|-----------------------|-------------------------------------------------------------------------------------------------------------------|
+| D-Pad                 | Move Focus / Navigation <br/>Tweak values (when activated with A) <br/> Resize Window (when holding X)            |
+| Left Stick            | Scroll <br> Move Window (when holding X)                                                                          |
+| X (Left Face Button)  | Tap: Toggle Menu<br>Hold + L1/R1: Focus Window<br>Hold + D-Pad: Resize Window<br />Hold + Left Stick: Move Window |
+| Y (Upper Face Button) | Exit text / on-screen keyboard                                                                                    |
+| B (Right Face Button) | Cancel / Close / Exit                                                                                             |
+| A (Lower Face Button) | Activate / Open / Toggle <br/>Tweak values with D-Pad<br/>(+ L1/R1 to tweak slower/faster)                        |
+
+If using the controller as a mouse then the following inputs are supported:
+
+| Input                 | Description               | 
+|-----------------------|---------------------------|
+| Left stick            | Move Mouse Pointer        |
+| A (Lower Face Button) | Left Mouse Button (Btn1)  |
+| B (Right Face Button) | Right Mouse Button (Btn2) |
+
+You can adjust the sensitivity using `imgui_ControllerMouseSensitivity`
+
+## Discrete input mode
+By default, when ImGui is visible input will be sent to both ImGui and the O3DE game/editor. Sometimes, it is more desirable to only control either ImGui or the game/editor at any given time. To facilitate this, the ImGui Gem supports a discrete input mode.
```

Comment:
Minor suggestion, using "By default" to open consecutive statements. One can be removed.

```suggestion
When ImGui is visible, input is sent to both ImGui and the O3DE game/editor. Sometimes, it is more desirable to only control either ImGui or the game/editor at any given time. To facilitate this, the ImGui Gem supports a discrete input mode.
```